internal/testrunner/runners/static/runner.go

```diff
@@ -72,7 +74,33 @@ func (r runner) run(ctx context.Context) ([]testrunner.TestResult, error) {
 		return result.WithError(fmt.Errorf("failed to read manifest: %w", err))
 	}
 
-	return r.verifySampleEvent(pkgManifest), nil
+	// join together results from verifyStreamConfig and verifySampleEvent
+	return append(r.verifyStreamConfig(ctx, r.options.PackageRootPath), r.verifySampleEvent(pkgManifest)...), nil
```

Comment:
We had the discussion in the past about where to start. If I remember correctly, we went for global first as it was easier to implement and kept the option open, that a single benchmark could be run for more then just one data stream. Like you generate a rally track for all k8s data in parallel, you don't test just one data stream at the time. 

Overall I share your sentiment, that the benchmark definition should be part of the dataset, but there could be global scenarios (likely even across integrations). The corpus tool had it fully disconnected which was not ideal either from my perspective.

Reply:
> a single benchmark could be run for more then just one data stream

Looking to the templates we have now, I think it would be pretty convoluted to create one to generate data suitable for multiple data streams, so even when they are at the root of the package, and run at the same time, they are specific to each data stream. I think we could have the definitions per data-stream, and still be able to run multiple of them at the same time.